pyproject.toml

```diff
@@ -49,12 +49,32 @@ path = "src/docker_snapshot/__about__.py"
 
 
 [tool.hatch.envs.types]
-extra-dependencies = ["mypy>=1.0.0"]
+extra-dependencies = ["mypy>=1.0.0", "types-docker"]
 
 [tool.hatch.envs.types.scripts]
 check = "mypy --install-types --non-interactive {args:src/docker_snapshot}"
 
 
+[tool.mypy]
+python_version = "3.9"
+strict = true
+warn_unused_configs = true
+warn_unused_ignores = true
+check_untyped_defs = true
+warn_redundant_casts = true
+warn_return_any = true
+warn_no_return = true
+warn_unreachable = true
+warn_incomplete_stub = true
+no_implicit_optional = true
+no_implicit_reexport = true
+disallow_any_explicit = true
+disallow_any_generics = true
+disallow_any_unimported = true
+disallow_subclassing_any = true
+disallow_untyped_defs = true
```

Comment:
Let me know if you disagree with any settings here, this is just a pretty reasonable default IMO. At least `strict` is pretty much non-negotiable though.

Reply:
I am not experienced enough to have an opinion yet, will take your word for it and change it if I disagree later :D